src/core/operations/Jq.mjs

```diff
@@ -0,0 +1,57 @@
+/**
+ * @author zhzy0077 [zhzy0077@hotmail.com]
+ * @copyright Crown Copyright 2023
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation.mjs";
+import OperationError from "../errors/OperationError.mjs";
+import jq from "jq-web";
```

Comment:
This import seems to increase the size of the bundle by around 3-4MB, including the size of "Code" modules from 700KB transferred to 4MB transferred.  I suspect there's not a lighter-weight way of doing this, but would it be possible to put it in it's own module?  The "Code" module is used by a bunch of other operators.